torch/_inductor/codegen/mps.py

```diff
@@ -141,6 +141,15 @@ def _print_ToFloat(self, expr: sympy.Expr) -> str:
         x = self.doprint(expr.args[0])
         return f"static_cast<float>({x})"
 
+    def _print_Float(self, expr: sympy.Expr) -> str:
+        if expr.is_integer:
+            # sympy considers 0.0 to be integer, but mps doesn't.
```

Comment:
Despite the name, MPS codegen simply emits Metal shaders. But I think it's a more generic statement for all C-like languages, so I wonder how it works for CPPInductor for example (or do they already have this logic somewhere?)
```suggestion
            # sympy considers 0.0 to be integer, but Metal doesn't.
```